etc/cvmfs/domain.d/desy.de.conf

```diff
@@ -1,10 +1,14 @@
-# Replace this file with a symlink to ../osghosted.conf once the selinux
-#  issue in https://sft.its.cern.ch/jira/browse/CVM-1922 is no longer a
-#  concern.
-
-# refer to all variables set by osghosted.conf so cvmfs will read them
+# These are here so cvmfs will notice them if common.conf sets them
 CVMFS_USE_CDN="$CVMFS_USE_CDN"
 CVMFS_HTTP_PROXY="$CVMFS_HTTP_PROXY"
 CVMFS_FALLBACK_PROXY="$CVMFS_FALLBACK_PROXY"
-CVMFS_SERVER_URL="$CVMFS_FALLBACK_PROXY"
-. ../osghosted.conf
+. ../common.conf
+
+if [ "$CVMFS_USE_CDN" = "yes" ]; then
+    CVMFS_SERVER_URL="http://s1desy-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1cern-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1ral-cvmfs.openhtc.io/cvmfs/@fqrn@"
```

Comment:
Just to be sure, are the DESY repositories actually being replicated to CERN and RAL?

Reply:
Yes.  They were already being used from there in egi.